aten/src/ATen/native/quantized/cpu/qnnpack/src/fc-dynamic-run.cc

```diff
@@ -0,0 +1,119 @@
+#include <pytorch_qnnpack.h>
+#include <qnnpack_func.h>
+#include <cstring>
+
+namespace qnnpack {
+struct q8gemm_dq_context {
+  size_t k;
+  size_t k_stride;
+  size_t n;
+  size_t n_stride;
+  const uint8_t* a;
+  size_t a_stride;
+  const uint8_t* packed_w;
+  const float* bias;
+  float* c;
+  size_t c_stride;
+  struct pytorch_qnnp_conv_dynamic_quantization_params quantization_params;
+  const pytorch_q8gemm_dq_ukernel_function ukernel;
+};
+
+static void compute_q8gemm_dq(
+    const struct q8gemm_dq_context context[1],
+    size_t group_index,
+    size_t pixel_index,
+    size_t mr_block_start,
+    size_t nr_block_start,
+    size_t group_range /* always 1 */,
+    size_t pixel_range,
+    size_t mr_block_size,
+    size_t nr_block_size)
+{
+  const size_t k = context->k;
+  const size_t k_stride = context->k_stride;
+  const size_t n = context->n;
+  const size_t n_stride = context->n_stride;
+  const uint8_t* a = context->a;
+  const size_t a_stride = context->a_stride;
+  const void* packed_w = context->packed_w;
+  float* c = context->c;
+  const size_t c_stride = context->c_stride;
+  const float* bias = context->bias;
+
+  context->ukernel(
+      mr_block_size,
+      nr_block_size,
+      k,
+      a + (pixel_index + mr_block_start) * a_stride + group_index * k,
+      a_stride,
+      (const void*) ((uintptr_t) packed_w + (nr_block_start + group_index * n_stride) * (k_stride * sizeof(uint8_t) + sizeof(int32_t))),
+      bias,
+      c + (pixel_index + mr_block_start) * c_stride + nr_block_start + group_index * n,
+      c_stride,
```

Comment:
I think this should be c_stride * sizeof(float).

Reply:
OK so I modified the kernels to not need this multiplication.  Please undo these changes that affect the c_stride.